### PR TITLE
CLDR-15524 doc: fix http to https

### PR DESCRIPTION
**Pull request by srl295**
- also fix one bad link broken in CLDR-15655

CLDR-15524

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by srl295** [comment]
> I don't think we should change to _exactly_ what is shown for some of these. I recommend to use https://www.unicode.org rather than https://unicode.org _sans_ www.

i can change https://unicode.org to https://www.unicode.org , will do

**Comment by srl295** [comment]
@rickmcgowan check now

**Review by rickmcgowan — APPROVED** [review]
Updates seem to be in order, thanks. I hope someone is running a link check when this is all done, to make sure all links resolve, especially the outside links to external (non Unicode) sites. :-)

**Review by pedberg-icu — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ CLDR-_____
 
 <!--
 Thank you for your pull request.
-Please see http://cldr.unicode.org/index/process for general
```
Comment (kachgold) [inline .github/pull_request_template.md:7]: .github/pull_request_template.md

**Review by kachgold — APPROVED** [review]

**Comment by macchiati** [comment]
*www.*xxxxxx.yyyy was the style 20 years ago. But no longer. What you see
is xxxxxx.yyyy.

Now, they both need to resolve to the same content in any event. But there
is no need to hold on to the Model T approach.

**Pull request merged**